docs/RunningOnDevice.md

```diff
@@ -150,7 +150,7 @@ $ react-native run-android
 
 > Hint
 >
-> You can also use the `React Native CLI` to generate and run a `Release` build (e.g. `react-native run-android --configuration Release`).
+> You can also use the `React Native CLI` to generate and run a `Release` build (e.g. `react-native run-android --config Release`).
```

Comment:
```
➜  HelloProject react-native run-android --help

  react-native run-android [options]
  builds your app and starts it on a connected Android emulator or device

  Options:

    -h, --help          output usage information
    --install-debug
    --root [string]     Override the root directory for the android build (which contains the android directory)
    --flavor [string]   --flavor has been deprecated. Use --variant instead
    --variant [string]
    --config [string]   Path to the CLI configuration file
```

and when I type `react-native run-android --configuration` show this error

```
➜  HelloProject react-native run-android --configuration

  error: unknown option `--configuration'
```

Here is my versions `react-native-cli`

```
➜  HelloProject react-native --version
react-native-cli: 1.2.0
react-native: 0.38.0
```